src/_gettsim_tests/test_aggregation_functions.py

```diff
@@ -11,6 +11,7 @@
     grouped_mean,
     grouped_min,
     grouped_sum,
+    sum_by_p_id,
```

Comment:
Pls make an issue reminding us to add and test the other `aggr_by_p_id` functions.